Entities/AHuman.cpp

```diff
@@ -3579,8 +3579,9 @@ void AHuman::Update()
                     if (m_ArmsState != THROWING_PREP/* || m_ThrowTmr.GetElapsedSimTimeMS() > m_ThrowPrepTime*/)
                     {
                         m_ThrowTmr.Reset();
-                        // Activate the device, light the fuse etc!
-                        pThrown->Activate();
+						if (!pThrown->ActivatesWhenReleased()) {
```

Comment:
Is this just github messing with the display of white spacing?

![image](https://user-images.githubusercontent.com/4422100/72895723-6a389000-3cf4-11ea-9852-e5b1ad60151e.png)

Reply:
It's a spaces vs tabs problem. Slowly swapping to tabs from spaces but can't just auto-format everything cause other stuff gets screwed up.
So in short, yes.